Extensions/mutualchecker.css

```diff
@@ -1,8 +1,16 @@
-.mutuals::after{
+.mutuals::after, .mutuals::before{
```

Comment:
shouldn't this just be `.mutuals` instead? any pseudo-elements should use their parent's css afaik

Reply:
If I were to use just mutuals, wouldn't it affect the text itself as well? We only want the before and after styled, iirc

Reply:
oh right. forgot i changed it to not be its own element, whoops